src/components/m-button/mixins/_button.scss

```diff
@@ -129,16 +129,21 @@
   padding-bottom: 11px;
 }
 
+
 @mixin button__icon() {
   @include icon($button__icon-width);
 
-  padding: 10px;
+  position: absolute;
+
+  top: 50%;
 
-  border-left: 1px solid rgba($color-prim-white, 0.2);
```

Comment:
@TheDadi 
Hmh, this is a variation of the style at the guidelines:
https://design.axa.com/web-guidelines/buttons-and-links#types-of-buttons

We should keep it, or at least make it optional